src/order/filter/at_top_bot.lean

```diff
@@ -267,6 +267,10 @@ lemma tendsto_at_top_add_nonneg_right (hf : tendsto f l at_top) (hg : ∀ x, 0 
   tendsto (λ x, f x + g x) l at_top :=
 tendsto_at_top_add_nonneg_right' hf (eventually_of_forall hg)
 
+lemma tendsto_at_top_add (hf : tendsto f l at_top) (hg : tendsto g l at_top) :
+  tendsto (λ x, f x + g x) l at_top :=
```

Comment:
```suggestion
  tendsto (f + g) l at_top :=
```

Reply:
I'm not sure which form is better. Many theorems use `λ x, f x + g x` for a historical reason: `simp` is used to fail to apply `pi.add_apply` because of badly defined `pi.*` algebraic instances. With `f + g` the statement looks better but whenever you do `have h := tendsto_at_top_add h₁ h₂` you probably get something like `(λ x, x^2) + (λ x, x^3)`.